redisearch/pool.go

```diff
@@ -65,3 +65,18 @@ func (p *MultiHostPool) Get() redis.Conn {
 	return pool.Get()
 
 }
+
+func (p *MultiHostPool) Close() (err error) {
```

Comment:
not a big fan of the named returned variable thing, unless we want to defer-change the value of it. no biggie, just personal preferences.

Reply:
I believe it simplifies the method. specifically after the last change. please revise last changes